cmsplugin_articles_ai/models/articles.py

```diff
@@ -187,6 +187,20 @@ def get_absolute_url(self):
         """
         return reverse("article", kwargs={"slug": self.slug})
 
+    def clone_relations(self, src_obj, dst_obj):
+        """
+        Implements cloning relations from article to another
+
+        :param src_obj: Article object where the relations are copied from
+        :param dst_obj: Article object where the relations are copied to
+        """
+        dst_obj.tags = src_obj.tags.all()
+
+        for attachment in src_obj.attachments.all():
+            attachment.pk = None
+            attachment.article = dst_obj
+            attachment.save()
```

Comment:
So you reference the same tags instead of cloning them, but with attachments, you clone the attachments.. When is this cloning actually done? At the time when a new draft is created?